src/main/antora/modules/ROOT/pages/index.adoc

```diff
@@ -68,19 +65,19 @@ used in this book.
 |===
 |Placeholder |Description |Default Value
 
-|`_as-install_` |Represents the base installation directory for GlassFish Server. | Installations on the Solaris operating system, Linux operating system, and Mac operating system:
+|`_as-install_` |Represents the base installation directory for GlassFish Server. | Installations on the Linux, UNIX, or macOS:
 
 `_user's-home-directory_/glassfish6/glassfish`
```

Comment:
@BalusC I'm afraid changing this will break references to the examples. I want to revisit this once we revisit the examples. I could add the standard "We are working on a fresh, updated Jakarta EE Tutorial. This section hasn’t yet been updated" disclaimer above just this section. WDYT?

Reply:
Check out the disclaimer I added...

Reply:
Ah right. That's ok.